pkg/apis/admissionregistration/doc.go

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 
 // Package admissionregistration is the internal version of the API.
 // AdmissionConfiguration and AdmissionPluginConfiguration are legacy static admission plugin configuration
-// InitializerConfiguration and ExternalAdmissionHookConfiguration is for the
+// InitializerConfiguration and ValidatingWebhookConfiguration is for the
```

Comment:
InitializerConfiguration, ValidatingWebhookConfiguration, and MutatingWebhookConfiguration... ?